core/services/ocr2/plugins/functions/integration_tests/internal/testutils.go

```diff
@@ -45,10 +46,14 @@ import (
 )
 
 var (
-	DefaultSecretsBytes  = []byte{0xaa, 0xbb, 0xcc}
-	DefaultSecretsBase64 = "qrvM"
-	DefaultArg1          = "arg1"
-	DefaultArg2          = "arg2"
+	DefaultSecretsBytes      = []byte{0xaa, 0xbb, 0xcc}
+	DefaultSecretsBase64     = "qrvM"
+	DefaultSecretsUrlsBytes  = []byte{0x01, 0x02, 0x03}
+	DefaultSecretsUrlsBase64 = "AQID"
+	// DefaultThresholdSecretsHex decrypts to the JSON string `{"0x0":"qrvM"}`
```

Comment:
Using which keys? The ones from the other file? Let's keep them together in one place.